### PR TITLE
User interface: `MORD` manage orders: add first mord acs

### LINKED ISSUE #16: Acceptance criteria for `7003-MORD` Manage my orders
## The Chore

To create a set of Acceptance criteria for Vega user interfaces for frontend end to end system tests to be written and run against, and reported on by [approbation](https://github.com/vegaprotocol/approbation) to assess test coverage (and therefore confidence).

This first cut of A/C for end to end tests should focus on ensuring the features and functionality we have today is trading ready.

## Tasks

- [ ] Write 'em
- [ ] get 'em reviewed
- [ ] get 'em merged
- [ ] Point QA at 'em

## Link to user interface A/C in specs repo and related PRs

https://github.com/vegaprotocol/specs/tree/master/user-interface

https://github.com/vegaprotocol/specs/pull/1139
https://github.com/vegaprotocol/specs/pull/1181
https://github.com/vegaprotocol/specs/pull/1223

### PR DESCRIPTION
**Pull request by campbellssource**
this adds content (acceptance criteria) to the Manage orders section.

- [readable preview](https://github.com/vegaprotocol/specs/blob/user-interface/Add-first-MORD-ACS/user-interface/7003-MORD-manage_orders.md)

closes: https://github.com/vegaprotocol/ux-and-visual-design/issues/16

### CODE REVIEW

File: user-interface/7003-MORD-manage_orders.md [hunk 1]
```diff
@@ -1 +1,202 @@
...
+  - `Active​`
+    - How much of the order is filled / remains unfilled
+    - How close the mark price is to order
+    - If this order was filled at the limit price what would what effect would it have on realized PnL 
+    - I may want to amend or cancel this order
+  - `Expired​`
+    - When did it expire
+    - How much was filled / remaining
+  - `Cancelled​`
+    - What canceled it? When did I cancel it TODO double check that "cancelled" only comes from user action
```
Comment (mattrussell36) [inline user-interface/7003-MORD-manage_orders.md:35]: Do you need to do or update this TODO and the one under PARKED?

File: user-interface/7003-MORD-manage_orders.md [hunk 2]
```diff
@@ -1 +1,202 @@
...
+- if the order is `Active` &amp; is part of a liquidity or peg shape: **must** **not** see an option to cancel the individual order
+  - **could** see a link to cancel shape
+
+... so I can understand the state of my orders and decide what to do next
+### Filters
+
+When looking at a list of orders, I...
+
+- **should** have the ability to see all orders regardless of status in one list
+- **should** have the ability to see only active &amp; parked orders TODO update this based on what happens to parked orders
```
Comment (mattrussell36) [inline user-interface/7003-MORD-manage_orders.md]: TODO needs updating

File: user-interface/7003-MORD-manage_orders.md [hunk 3]
```diff
@@ -1 +1,202 @@
...
+  - **should** be able to add a secondary sort. e.g. by market then by date  
+- **should** have the default sorted by created time (or updated time if newer), with newest at the top
+- **should** retain sorting preferences between switching views / browser reload
+
+... so the sort order matches my mental model and or makes it easy for me to find the orders I am looking for  
+### Grouping
+
+When looking at a list of orders, I...
+
+- **should** be able to group orders by any field e.g. by market, lp etc
```
Comment (mattrussell36) [inline user-interface/7003-MORD-manage_orders.md:104]: What do you mean by grouping here. I see this as the same as sorting

File: user-interface/7003-MORD-manage_orders.md [hunk 4]
```diff
@@ -1 +1,202 @@
...
+- **should** have default grouping by market
+
+... so I can make decisions easily on a set of orders 
+## Cancel orders
+
+When looking to take order out of the book, I...
+
+- **must** select weather to cancel an individual order or all orders on a market (<a name="7003-MORD-009" href="#7003-MORD-009">7003-MORD-009</a>)
```
Comment (mattrussell36) [inline user-interface/7003-MORD-manage_orders.md]: weather -> whether

**Review by mattrussell36 — COMMENTED** [review]
There's a couple larger sections that still have TODO. Presumably thats intended. Left some comments

File: user-interface/7003-MORD-manage_orders.md (hunk 1 above)
Comment (campbellssource) [reply user-interface/7003-MORD-manage_orders.md:35]: resolved in : https://github.com/vegaprotocol/specs/pull/1334/commits/2ded0c5ae2a5f0dd4975dc46c47202660511d9f7 it doesn't change the fact we should link to the shape when a order is parked

File: user-interface/7003-MORD-manage_orders.md (hunk 2 above)
Comment (campbellssource) [reply user-interface/7003-MORD-manage_orders.md]: resolved in : https://github.com/vegaprotocol/specs/pull/1334/commits/01f208d055c69bb4b89a6e4cb0b84d655259886e should be able to filter by parked and active

File: user-interface/7003-MORD-manage_orders.md (hunk 4 above)
Comment (campbellssource) [reply user-interface/7003-MORD-manage_orders.md]: https://github.com/vegaprotocol/specs/pull/1334/commits/2876d62d2edc384802fcd2f8550004fa239f1cc3

File: user-interface/7003-MORD-manage_orders.md (hunk 3 above)
Comment (campbellssource) [reply user-interface/7003-MORD-manage_orders.md:104]: I mean grouping and sorting in the same way that excel does: https://edu.gcfglobal.org/en/excel2007/sorting-grouping-and-filtering-cells/1/

So I group all order by market. then sort the groups (maybe by expiry), with a secondary sort be price.

This way I can see all order by market (how I think about them), and then the ones with the in an order that makes sense

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**